src/pages/classes/classesAddEdit/ClassesAddEditView.tsx

```diff
@@ -2,89 +2,156 @@ import { ReactElement } from 'react';
+import { Button, createStyles, FormControl, InputLabel, makeStyles, MenuItem, Select, Theme } from '@material-ui/core';
+import { TextField } from 'formik-material-ui';
+import React from 'react';
+import { DateTimePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import { Save, Undo } from '@material-ui/icons';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      '& .MuiTextField-root': {
+        margin: theme.spacing(1),
+        width: '25ch',
+      },
+    },
+    form: {
+      display: 'flex',
+      flexDirection: 'column',
+      maxWidth: '500px'
+    },
+    formControl: {
+      margin: theme.spacing(1),
+      minWidth: 120,
+    },
+    selectEmpty: {
+      marginTop: theme.spacing(2),
+    },
+    button: {
+      margin: theme.spacing(1),
+    },
+  }),
+);
 
 export const ClassesAddEditView = ({
+  isEdit,
   locations,
   classTypes,
   foundClass,
   onSubmit,
 }): ReactElement => {
+  const classes = useStyles();
+
   return (
     <div
       style={{
         padding: 10,
       }}
     >
-      <h1>Classes Edit</h1>
-      <Link to="/classes">
-        <button>Back</button>
+      <h1>Classes {isEdit ? 'Edit' : 'Add'}</h1>
+      <Link to="/classes" style={{ textDecoration: 'none' }}>
+        <Button
+          variant="contained"
+          color="secondary"
+          className={classes.button}
+          startIcon={<Undo />}
+        >
+          Back
+        </Button>
       </Link>
-      <Formik initialValues={foundClass} onSubmit={onSubmit}>
-        {({ values, setFieldValue }) => (
-          <Form
-            style={{
-              display: 'flex',
-              flexDirection: 'column',
-            }}
-          >
-            <label>
-              Class Code:
-              <Field name="classCode" />
-            </label>
-            <label>
-              Period Start:
+      <MuiPickersUtilsProvider utils={LuxonUtils}>
+        <Formik
+          className={classes.root}
+          initialValues={foundClass}
+          onSubmit={onSubmit}>
+          {({ values, setFieldValue }) => (
+            <Form className={classes.form}
+            >
+
               <Field
+                component={TextField}
+                name="classCode"
+                label="Class Code"
+                required
+                variant="outlined"
+                className={classes.formControl}
+              />
+              <DateTimePicker
+                ampm={false}
+                format="dd/MM/yyyy HH:mm"
+                label="Period Start"
+                inputVariant="outlined"
                 name="periodStart"
-                render={() => (
-                  <Datetime
-                    onChange={(e) => setFieldValue('periodStart', e)}
-                    value={values.periodStart}
-                  />
-                )}
+                value={values.periodStart}
+                onChange={ (e) => setFieldValue("periodStart", e) }
```

Comment:
All onChange should look like this `onChange={e => setFieldValue("periodStart", e)}` or 
````onChange={e => (
  setFieldValue("periodStart", e)
)}````